### ISSUE TITLE
Support scaling in CTkCanvas 

### ISSUE THREAD

**Issue by briemla** [TomSchimansky/CustomTkinter#1742]
The current implementation of `CTkCanvas` only inherits from `tkinter.Canvas`. Other classes inherit from `CTkScalingBaseClass` which provides support for automatic scaling on different platforms. This would also be nice for `CTkCanvas` elements.

**Comment by TomSchimansky** [comment]
Yeah that would be nice of course, but I don't know if it's even possible. I think it would be a lot of work because the canvas has so many methods that need to be supported, I don't know if I will ever try to do it.
